### PR TITLE
Issue 93

### PR DESCRIPTION
**Pull request by BasThomas**

### CODE REVIEW

File: _drafts/2017-11-02-issue-93.md [hunk 1]
```diff
@@ -9,50 +9,60 @@ sponsor:
...
+In Signapore, the government is learning [seniors to write Swift](https://www.cnet.com/news/singapore-teaches-its-seniors-to-code/) for the Hour of Code.
 
-> TODO
+Slava's fingers are itching to [implement an `Either` type](https://twitter.com/slava_pestov/status/925133908835835904). Soon™?
 
-### Rejected proposals
+Chris Bailey announced [Kitura 2.0](https://developer.ibm.com/swift/2017/10/30/kitura-20/), IBM's web framework for building Swift applications.
 
-> TODO
+Paul Hudson introduced the Swift Community Awards a few weeks ago, and [the shortlist](https://www.hackingwithswift.com/awards) is available. It includes **this very newsletter** as well as it's podcast, [Swift Unwrapped](https://spec.fm/podcasts/swift-unwrapped). That's quite the honor! Thanks for all of you who mentioned us, and feel free to vote and make it even more awesome! 😎
```
Comment (BasThomas) [inline _drafts/2017-11-02-issue-93.md]: Wasn't entirely sure about this one - I think we want it in the introduction maybe? @jessesquires, what do you think?

Comment (jpsim) [reply _drafts/2017-11-02-issue-93.md]: s/it's/its/

Comment (BasThomas) [reply _drafts/2017-11-02-issue-93.md]: 😬

File: _drafts/2017-11-02-issue-93.md [hunk 2]
```diff
@@ -9,50 +9,60 @@ sponsor:
...
+Video's of JP's and my talk from FrenchKit are available. JP talked about [Profiling Swift Performance on Linux](https://www.youtube.com/watch?v=TWeLLTFjqXg), I gave a talk about [What's up with Swift 5](https://www.youtube.com/watch?v=XXqZaKodLfA).
 
-### Accepted proposals
+Slava shared a gentle reminder of the awesome [`swift-source-compat-suite`](https://twitter.com/slava_pestov/status/924133915199119360), where you can add your open source project to make sure new additions to Swift do not break source compatability.
 
-> TODO
+The first swift evolution proposals ([SE-0075](https://github.com/apple/swift-evolution/blob/master/proposals/0075-import-test.md) and [SE-0186](https://github.com/apple/swift-evolution/blob/master/proposals/0186-remove-ownership-keyword-support-in-protocols.md)) have been implemented in Swift 4.1, so this Swift version has been added to the [status page](https://apple.github.io/swift-evolution/) now, [too](https://github.com/apple/swift-evolution/pull/755).
 
-### Returned proposals
+In Signapore, the government is learning [seniors to write Swift](https://www.cnet.com/news/singapore-teaches-its-seniors-to-code/) for the Hour of Code.
```
Comment (jpsim) [inline _drafts/2017-11-02-issue-93.md]: s/learning/teaching/

File: _drafts/2017-11-02-issue-93.md [hunk 3]
```diff
@@ -9,50 +9,60 @@ sponsor:
...
+>
+> If we do this, the vast majority of the Python ecosystem should be directly usable from within Swift code, and there are only a few major syntactic differences (e.g. ranges work differently). We would add failable inits to the primitive datatypes like `Int`/`String`/etc to convert `Python.Object` values into them, and add the corresponding non-failable conversions from `Python.Object` to those primitives.
+>
+> Overall, I think it will provide a really nice experience, and allow us to leverage the vast majority of the Python ecosystem directly in Swift code. This project would also have much more narrow impact on the Swift compiler than the ObjC importer (since it works completely differently). For a first cut, I don’t think we would have to worry about Swift classes subclassing Python classes, for example.
 
 ### Finally
 
-> TODO: something funny/fun. tweet, link, etc.
+Joe Groff's human was [very, very prepared](https://twitter.com/jckarter/status/925122252479148032) for his talk at Swift Summit.
```
Comment (jessesquires) [inline _drafts/2017-11-02-issue-93.md]: > Joe Groff's human

... 🤔 did you mean dog? 🐶 😄

**Comment by jessesquires** [comment]
awesome issue! 🎉

**Pull request merged**